config/dev.exs

```diff
@@ -22,6 +22,10 @@ config :maru, MongoosePush.Router,
     otp_app: :mongoose_push
   ]
 
+config :mongoose_push, MongoosePush.Service,
```

Comment:
We can put this in the "global" `config/config.exs` and just override it in test env. No need to duplicating it in both prod and dev.